src/cloud-api-adaptor/libvirt/kcli_cluster.sh

```diff
@@ -62,6 +62,11 @@ create () {
 			-P multus=false \
 			-P autolabeller=false "
 	fi
+	if [[ ${TARGET_ARCH} == "aarch64" ]]; then
```

Comment:
minor nit: Could we make this an `elif` to be a bit neater?

Reply:
Sure, will do. Thanks.